packages/itwinui-react/src/core/RadioTiles/RadioTile.tsx

```diff
@@ -28,29 +57,82 @@ type RadioTileProps = {
  * <RadioTile label='My tile' description='Some info' icon={<SvgSmileyHappy />} />
  */
 export const RadioTile = React.forwardRef((props, ref) => {
-  const { icon, label, description, className, style, ...rest } = props;
+  const {
+    icon,
+    label,
+    style,
+    description,
+    setFocus = false,
+    tileProps = {},
+    inputProps = {},
+    contentProps = {},
+    iconProps = {},
+    labelProps = {},
+    subLabelProps = {},
+  } = props;
 
   const inputElementRef = React.useRef<HTMLInputElement>(null);
   const refs = useMergedRefs<HTMLInputElement>(inputElementRef, ref);
 
+  React.useEffect(() => {
+    if (inputElementRef.current && setFocus) {
+      inputElementRef.current.focus();
+    }
+  }, [setFocus]);
```

Comment:
why did you add this `useEffect` and the `setFocus` prop? they were intentionally removed in #1406.

Reply:
I think I added that by mistake, I'll remove it.

Reply:
It has been removed

Reply:
needs to be removed from the `type` as well

Reply:
done